lms/djangoapps/shoppingcart/models.py

```diff
@@ -1746,6 +1746,8 @@ def purchased_callback(self):
         self.course_enrollment.activate()
 
     def additional_instruction_text(self):
+        if not self.course_enrollment.is_verified_enrollment():
```

Comment:
Your build is [failing quality](https://build.testeng.edx.org/job/edx-platform-test-subset/13715/console) because of this line.